src/infra/src/table/migration/m20241220_000400_populate_users_table.rs

```diff
@@ -0,0 +1,185 @@
+// Copyright 2024 OpenObserve Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+use config::{
+    ider,
+    meta::user::{DBUser, UserRole},
```

Comment:
Can we copy these type definitions into this migration and reference those from within inside this module? My concern is that if data types in `config::meta::user` change in the future then the behavior of this migration will change and potentially have different behavior for different users depending on when they run the migration

Reply:
Makes sense!